packages/create-remix/templates/arc-stack/server/config.arc

```diff
@@ -0,0 +1,5 @@
+@aws
+# runtime nodejs14.x
+# memory 1152
+# timeout 30
+# concurrency 1
```

Comment:
Technically, since these settings are commented out (with the leading `# `) this file doesn't do anything. It can safely be removed.

It does demonstrate some of Arc's [function configuration options](https://arc.codes/docs/en/reference/configuration/function-config) which take priority over the `@aws` entry in the project's `app.arc` file.

Up to you if you want to include it in the stack template, I just wanted to point out that it isn't needed.